code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1057,6 +1058,25 @@
 	REMOVE_TRAIT(M, TRAIT_STABLEHEART, type)
 	REMOVE_TRAIT(M, TRAIT_STABLELIVER, type)
 	..()
+/datum/reagent/medicine/penthrite
+	// Same as corazone. but slightly better
+	name = "Penthrite"
+	description = "An explosive compound used to stabilize heart conditions. May interfere with stomach acid!"
+	color = "#F5F5F5"
+	self_consuming = TRUE
+
+/datum/reagent/medicine/penthrite/on_mob_metabolize(mob/living/M)
+	..()
+	if(prob(10))
+		M.adjustOrganLoss(ORGAN_SLOT_STOMACH,-REM)
```

Comment:
is this suppose to heal...

I'd also rather it give low constant damage than rng. Eitherway should be a C2.

Reply:
> is this suppose to heal...
> 
> I'd also rather it give low constant damage than rng. Eitherway should be a C2 (if it's suppose to heal this is mega cringe).

nah no healing, oranges already told me that. this is bascially shittier corazone, i might add more functionality later. And yeah i agree ill make it very low constant damage

Reply:
wait fuck i see the problem